src/components/Header/Header.js

```diff
@@ -29,7 +48,30 @@ export const Header = () => {
         </header>
         <SideMenu />
       </div>
-  )
+    )
+  }
+}
+
+const mapStateToProps = (state) => {
+	    return {
+        cropImageName: state.cropImageName,
+        cycleFlag: state.cycleFlag
+        }
+}
+
+const cycleThrough = (cropImageName) => {
+	return {type: "CHANGE_CROP_IMAGE", payload: cropImageName }
+}
+
+const toggleCycleFlag = () => {
+	return {type: "TOGGLE_CYCLE_FLAG" }
+}
+
+
+const mapDispatchToProps = (dispatch) => {

```

Comment:
To make this a bit more readable you can ditch this entire function and replace line 77 with 
```
export default connect(mapStateToProps, {cycleThrough, toggleCycleFlag})(Header)
```
The code will do pretty much the same thing since the mapDispatchToProps can be a function (like how you're doing it) or it can be an object which will have its properties automapped to dispatch.